controllers/links.js

```diff
@@ -0,0 +1,43 @@
+var express = require("express");
+var db = require("../models");
+var router = express.Router();
+
+var Hashids = require("hashids");
+var hashids = new Hashids("mine is sea salt");
+////////////////////////
+
+// Get long URL from user
+router.post("/", function (req, res) {
```

Comment:
This is where you should be creating the database entry. I had a look through your code and I can't see where you create entries. You can try doing something like: 
`findOrCreate({
  where: { url: req.body.url }
})`

and then instead of using a .catch you should use a .spread

'.spread(function(){})`